decidim/decidim
# Implement etiquette validator on meetings

Meeting can be updated with an empty body

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12712 
- Related to #12713  
- Related to #12714 

#### Testing
 - Enable rich text editor for participants from Organization settings
 - Create a new Meeting 
 - Play with the `Title` and `Description` fields content
 - Form save should be prevented by the following error messages or combination, depending on the content:
    - `can't be blank`, 
    - `must start with a capital letter`
    - `is using too many capital letters (over 25% of the text)`
 - the same behavior should be observable for the `Registration term` field, if the `Registration type` is set to 'On this platform'

### :camera: Screenshots
![Description](URL)
![Screenshot 2024-05-23 133049](https://github.com/decidim/decidim/assets/150658895/d8900cee-378d-49f3-a2a7-07b4f3f6e00b)
![Screenshot 2024-05-23 133125](https://github.com/decidim/decidim/assets/150658895/1f6e6ddf-4eab-4c80-88d5-fcecf0e681ac)
![Screenshot 2024-05-23 133436](https://github.com/decidim/decidim/assets/150658895/b043f4be-a7b9-4aea-b61e-be1a34552e5e)

:hearts: Thank you!
b2b2-74e3dff22590)
